ui/ops/src/dynamic/widgets/meter/MeterHistoryCard.vue

```diff
@@ -24,6 +24,16 @@
               </v-list-item>
               <v-list-subheader title="Data"/>
               <period-chooser-rows v-model:start="_start" v-model:end="_end" v-model:offset="_offset"/>
+              <v-list-subheader title="Metric" v-if="Object.keys(props.scaleValues).length > 0"/>
+              <v-list-item v-if="Object.keys(props.scaleValues).length > 0">
+                <v-btn-toggle
+                    mandatory v-model="metricType"
+                    variant="outlined" density="compact"
+                    divided class="ml-4">
+                  <v-btn :value="'unscaled'" size="small" :text="unit"/>
+                  <v-btn :value="'scaled'" size="small" :text="props.scaleUnit"/>
+                </v-btn-toggle>
+              </v-list-item>
```

Comment:
I think a single line would look better than header+buttons. Just like we do for the period chooser. Also I think the scaled button should have a `/` in front of the scaleUnit:

```
{ Metric          [  kWh  |  /m²  ] }
```